SPECS/glusterfs/glusterfs.spec

```diff
@@ -753,16 +1071,16 @@ fi
 # add marker translator
 # but first make certain that there are no old libs around to bite us
 # BZ 834847
-if [ -e %{_sysconfdir}/ld.so.conf.d/glusterfs.conf ]; then
-    rm -f %{_sysconfdir}/ld.so.conf.d/glusterfs.conf
+if [ -e /etc/ld.so.conf.d/glusterfs.conf ]; then
```

Comment:
Please use rpm macros wherever possible. E.g. %{_sysconfdir} in place of /etc

Reply:
this is from the upstream spec.in ... see previous comment about aligning to that file.